tasks/elasticsearch-config.yml

```diff
@@ -86,8 +79,8 @@
 - name: Delete Default Logging File
   file: dest=/etc/elasticsearch/logging.yml state=absent
 
-- name: Delete Default Logging File (5.x)
+- name: Delete Default Logging File
   file: dest=/etc/elasticsearch/log4j2.properties state=absent
 
-- name: Delete Default JVM Options File (5.x)
+- name: Delete Default JVM Options File
```

Comment:
Well spotted. The start script doesn't seem to take from the config directory but relies on the ES_HOME/conf. I don't see the service not starting though - it may explain the 4 failed tests though.
It seems the new start scripts in the 5.x package actually export this, similar to what @alessandroniciforo  has done here. https://github.com/elastic/ansible-elasticsearch/pull/214/files
Rather than inserting custom logic i think we should update the start scripts and insert the export of ES_JVM_OPTIONS into the defaults file.